profiles/permissions.restrictive

```diff
@@ -4,31 +4,24 @@
 # Copyright (c) 2019 SUSE Software Solutions Germany GmbH, Germany.
 #
 # Author: Roman Drahtmueller <draht@suse.de>, 2001
+# Author: Matthias Gerstner <matthias.gerstner@suse.de>, 2019
 #
-# 
-# See /etc/permissions for general hints on how to use this file.
+# See /etc/permissions and `man 5 permissions` for general hints on how to use
+# this file.
 #
-# /etc/permissions.paranoid is NOT designed to be used in a single-user as
-# well as a multi-user installation, be it networked or not.
+# This permissions profile has most sgid and suid bits cleared. As a result
+# parts of the system may become unusable for non-privileged users. In
```

Comment:
Do we want to explicitly solicit bug reports about (completely) broken functionality here? (Unsure myself)

Reply:
Could become difficult to explain that we only accept bug reports about broken "base functionality", whatever that is. I'd not add any extra motivation there.